src/Test/WordPress/HooksTest.php

```diff
@@ -42,9 +42,10 @@ public function setup()
         $this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
 
         $this->hooks = $this->getMockBuilder('\CF\WordPress\Hooks')
-            ->setMethods(array('setHooks'))
+            ->disableOriginalConstructor()
+            ->setMethods(array('__construct')) // This is a hack to make the tests work
```

Comment:
Oh wait this is the HookTest, why are we mock it at all?  If this is the class under test it should be real.